web/client/components/geostory/common/map/LocationListItemEditable.jsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import TitleEditable from '../../builder/TitleEditable';
+import GeneralButton from '../../../misc/GeneralButton';
+const {Glyphicon} = require('react-bootstrap');
+const enhanceTooltip = require('../../../misc/enhancers/tooltip');
+const Glyph = enhanceTooltip(Glyphicon);
+
+const LocationItemEditable = (props) => {
```

Comment:
If we need a list of card we could use components/misc/cardgrids/SideCard or components/misc/cardgrids/SideGrid instead of creating a new component